### ISSUE TITLE
How can you achieve ADE/FDE scores that are lower than the sensor noise of the recording system?

### ISSUE THREAD

**Issue by kav-hareichert** [ViviXiang/SocialCVAE#3]

**Comment by F-Nagisa** [comment]
In test phase, the code use the ground true position to embed the semantic map rather than predicted position. This may improve the performance.
See line 213-219 in train_SDD.py

**Comment by 18821252916** [comment]
I agree with F-Nagisa. Also, some recent work has used real coordinates of neighbors/obstacles, etc., which seems to improve prediction accuracy, and I am skeptical of such results.
